spark/src/main/resources/interpreter-setting.json

```diff
@@ -77,6 +77,12 @@
         "propertyName": "zeppelin.spark.maxResult",
         "defaultValue": "1000",
         "description": "Max number of SparkSQL result to display."
+      },
+      "zeppelin.spark.importImplicit": {
+        "envName": "ZEPPELIN_SPARK_IMPORTIMPLICIT",
+        "propertyName": "zeppelin.spark.importImplicit",
```

Comment:
@felixcheung Do you have any comments on this? 

I was thinking and decided to not to change the name of the option, even despite the fact it is too broad. Reason: we probably do not want to maintain separate options to load different sets of implicits (this feature is mainly for disabling all implicits and importing as needed), so I am okay with current implementation.